test/Interpreter/enforce_exclusive_access.swift

```diff
@@ -306,7 +306,7 @@ ExclusiveAccessTestSuite.test("SequentialKeyPathWritesDontOverlap") {
 
   c[keyPath: getF] = 7
   c[keyPath: getF] = 8 // no-trap
-  c[keyPath: getF] += c[keyPath: getF] + 1 // no-trap
+  c[keyPath: getF] += c[keyPath: getF] // no-trap
```

Comment:
Question. Was this filecheck test changed for a reason.

Reply:
Cleanup in a closely related test case.